src/components/dialogs/edit-package-details-dialog.tsx

```diff
@@ -86,30 +106,27 @@ export const EditPackageDetailsDialog = ({
         package_id: packageId,
         description: formData.description,
         website: formData.website,
+        is_private: formData.visibility == "private",
       })
-
-      if (response.status !== 200) {
-        console.error("Failed to update package details:", response.data)
+      const privacyUpdateResponse = await axios.post("/snippets/update", {
+        snippet_id: packageId,
+        is_private: formData.visibility === "private",
```

Comment:
this may be required

```suggestion
        is_private: formData.visibility === "private",
        is_public: formData.visibility === "public",
```